src/Symfony/Component/Validator/Constraints/BackedEnumValue.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+/**
+ * Validates that a backed enum can be hydrated from a value.
+ *
+ * @author Aurélien Pillevesse <aurelienpillevesse@hotmail.fr>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class BackedEnumValue extends Constraint
```

Comment:
this constraint needs to have the `HasNamedArguments` applied on it to be compatible with the XML and Yaml mapping loaders, as its constructor does not support the legacy signature providing an array `$options` but only supports the new way with named arguments.
This attributes is what makes those loaders use the new way.

Reply:
Fixed